core/src/main/scala/org/apache/spark/ui/jobs/StagePage.scala

```diff
@@ -501,25 +499,21 @@ private[ui] class TaskPagedTable(
 
   override val dataSource: TaskDataSource = new TaskDataSource(
     stage,
-    currentTime,
```

Comment:
Similar to the comment for AllJobsPage, can we remove `currentTime` from `TaskPagedTable`?